client/lib/feed-stream-store/index.js

```diff
@@ -21,15 +20,15 @@ function feedKeyMaker( post ) {
 	return {
 		feedId: post.feed_ID,
 		postId: post.ID,
-		localMoment: moment( post.date ),
+		date: new Date( post.date ),
```

Comment:
i don't really know why we were using moment in the first place. we just need a date.